akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -58,7 +58,11 @@ object ClusterClientSettings {
       refreshContactsInterval = config.getDuration("refresh-contacts-interval", MILLISECONDS).millis,
       heartbeatInterval = config.getDuration("heartbeat-interval", MILLISECONDS).millis,
       acceptableHeartbeatPause = config.getDuration("acceptable-heartbeat-pause", MILLISECONDS).millis,
-      bufferSize = config.getInt("buffer-size"))
+      bufferSize = config.getInt("buffer-size"),
+      reconnectTimeout = config.getString("reconnect-timeout") match {
+        case "off" ⇒ None
```

Comment:
Thought about it for a bit, I like that it is clear that it is disabled rather than not defined - there is _no_ timeout. If you feel it is important I'll change it.